src/WebJobs.Extensions.NotificationHubs/NotificationHubAttribute.cs

```diff
@@ -39,12 +39,14 @@ public sealed class NotificationHubAttribute : Attribute
         /// Optional. A string value indicating the app setting to use as the Notification Hubs connection
         /// string, if different than the one specified in the <see cref="NotificationHubsConfiguration"/>.
         /// </summary>
+        [AutoResolve(AllowTokens = false)]
         public string ConnectionStringSetting { get; set; }
 
         /// <summary>
         /// Optional. The Notification Hub Name to use, if different than the one specified in the
         /// <see cref="NotificationHubsConfiguration"/>.
         /// </summary>
+        [AutoResolve(AllowTokens = false)]
```

Comment:
Pragna and I were IMing -- she actually doesn't want [AutoResolve] at all on this property. She'll pull the default fallback from app settings using the name resolver, but everything else will use a literal.

---
In reply to: [70493194](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/120#discussion_r70493194) [](ancestors = 70493194,70492852)